airbyte-server/src/main/java/io/airbyte/server/handlers/ArchiveHandler.java

```diff
@@ -157,7 +157,7 @@ private void checkImport(Path tempFolder) throws IOException, JsonValidationExce
 
   private Optional<UUID> getCurrentCustomerId() {
     try {
-      return Optional.of(configRepository.getStandardWorkspace(PersistenceConstants.DEFAULT_WORKSPACE_ID).getCustomerId());
+      return Optional.of(configRepository.getStandardWorkspace(PersistenceConstants.DEFAULT_WORKSPACE_ID, false).getCustomerId());
```

Comment:
same here. i think we want true. we want the default workspace to be returned even if deleted here.